primer/brand
# Created a Medium Button Style

## Summary

In this PR I have created the medium button style for the button component. I have also updated the hero component to use the large button style as it was intended.

![image](https://user-images.githubusercontent.com/31220082/190152666-811d9c5f-7320-40e5-a672-2294f239e98f.png)


## List of notable changes:

- **added** medium button style for the button component
- **updated** hero component to use the large button style

## What should reviewers focus on?

Making sure that the Button component's new style meets the spec provided.

## Steps to test:

1. Open the Button component in CI-deployed preview environment
2. Go to Button story in Storybook
3. Verify that Button has two configurable states for size 'large' and 'medium' with 'medium' being the default
4. Check that the Hero component is using the large button style

## Supporting resources (related issues, external links, etc):

- [Issue](https://github.com/github/primer/issues/1093)
- [Spec](https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=373%3A8028)

## Contributor checklist:


## Reviewer checklist:
